library/errata_tool_product.py

```diff
@@ -264,6 +286,9 @@ def html_form_data(client, params):
     data['product[move_bugs_on_qe]'] = int(params['move_bugs_on_qe'])
     text_only_dists = int(params['text_only_advisories_require_dists'])
     data['product[text_only_advisories_require_dists]'] = text_only_dists
+    exd_org_group = params['exd_org_group']
```

Comment:
Once we default `exd_org_group` to `None`, we'll check for `None` here (like we do for `docs_reviewer` above), and only POST `exd_org_group_id` if we have a value.

```suggestion
    exd_org_group = params['exd_org_group']
    if exd_org_group is not None:
        exd_org_group_id = int(EXD_ORG_GROUPS[exd_org_group])
        data['product[exd_org_group_id]'] = exd_org_group_id    
```

Reply:
You're right, good catch.